tst/integration/test_json_basic.py

```diff
@@ -180,8 +180,12 @@ def test_sanity(self):
 
         assert b'OK' == client.execute_command(
             'JSON.SET', k2, '.', '[1,2,3,4,5]')
-        assert [b'{"a":"1","b":"2","c":"3"}', b'[1,2,3,4,5]'] == client.execute_command(
+        assert b'OK' == client.execute_command(
```

Comment:
We need more test coverage:
1. unit test to tst/unit/dom_test.cc to test dom_verify_value.
2. add a new pytest function for json.mset and cover various cases:
* just one key
* multiple keys
* root path
* non-root path
* multiple keys with mixed root path and non-root paths
* various types of json value arg: object, array, string, number, bool